releases/v20.2.0-alpha.2.md

```diff
@@ -0,0 +1,350 @@
+- Previously, issuing a `SIGTERM` signal twice or after another signal initiated a hard shutdown for a node. Now the first `SIGTERM` signal initiates a graceful shutdown and further occurrences of `SIGTERM` are ignored. To initiate a hard shutdown, issue `SIGINT` two times (or issue a `SIGINT` signal once after a `SIGTERM` signal). [#50539][#50539]
+- Specifying the same option multiple times in the `WITH` clause of the [`BACKUP`](../v20.2/backup.html) statement now results in an error message. Additionally, quoted option names are not allowed anymore. [#50723][#50723]
+
+### Security updates
+
+- HTTP endpoints beginning with `/debug/` now require a valid [admin](../v20.2/authorization.html) login session. [#50483][#50483]
+
+### General changes
+
+- Go 1.14 is now the minimum required version necessary to build CockroachDB. [#50671][#50671]
+- The docker image available with CockroachDB now includes the GEOS files required to work with Geospatial types in CockroachDB. [#50362][#50362]
+- Schema changes are now logged in greater detail. [#50372][#50372]
+- Added new options to YCSB to mirror OpenSource YCSB options. [#50546][#50546]
+- Links returned in error messages to point to unimplemented issues now use the CockroachLabs redirect/short-link server. [#49836][#49836]
+
+### Enterprise edition changes
+
+- The default flush interval for [changefeeds](../v20.2/change-data-capture.html) that do not specify a `RESOLVED` option is now 5s instead of 200ms to more gracefully handle higher-latency sinks. [#49770][#49770]
+- [Full cluster restore](../v20.2/restore.html#full-cluster) is now more resilient to transient transaction retry errors while it restores. [#49950][#49950]
+- [`RESTORE`](../v20.2/restore.html) now remaps user-defined types within the backup to existing types in the cluster. [#50349][#50349]
+- `BACKUP` can now run in `detached` mode, which means the Job ID is now returned immediately without waiting for the `BACKUP` job to finish. [#50775][#50775]
+
+### SQL language changes
+
+- For Postgres compatibility, CockroachDB now supports the `ANALYZE <tablename>` statement that collects statistics on the given table for use by the [optimizer](../v20.2/cost-based-optimizer.html). The statement is functionally equivalent to the existing [`CREATE STATISTICS`](../v20.2/create-statistics.html) statement. [#49816][#49816]
+- Implemented the `ST_Transform`, `ST_Intersection`, `ST_PointOnSurface`, and `ST_Union`, `ST_Segmentize({geometry, float8})`, `ST_Project({geography,float8,float8})`, `ST_LongestLine({geometry, geometry})`, `ST_Azimuth({geometry,geometry})`, `ST_ShortestLine({geometry, geometry})`, `ST_Centroid ({geometry,bool})` functions. [#49783][#49783], [#49833][#49833], [#49827][#49827], [#49949][#49949], [#50018][#50018], [#50188][#50188], [#50292][#50292], [#50484][#50484]
```

Comment:
Suggest updating this to use approximately the same language as the next bullet:

> Implemented the following builtins: ...